web/src/BuildButton.test.tsx

```diff
@@ -219,4 +232,18 @@ describe("SidebarBuildButton", () => {
       expectWithTooltip(button, BuildButtonTooltip.AlreadyQueued)
     })
   })
+
+  describe("stop builds", () => {
+    it("renders a stop button when the build is in progress", () => {
+      const root = mount(<BuildButtonTestWrapper isBuilding={true} />)
```

Comment:
1) whoops, none of the existing tests had isBuilding=true, so this change didn't break any existing tests
2) I feel like there should be more tests here, but I didn't come up with any. All the underlying stuff is just standard ApiButton logic that we probably shouldn't re-test.